pipeline/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM public.ecr.aws/lambda/python:latest
+
+WORKDIR ${LAMBDA_TASK_ROOT}
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY live_pipeline.py .
+COPY extract.py .
+COPY transform.py .
+COPY load.py .
+COPY prompts.py .
+COPY judge_matching.py .
+COPY judges_seed.py .
+
+EXPOSE 5432
```

Comment:
This is definetely all the ports we need?

Reply:
Hi, 
Great spot Miles, I've looked into this and saw that you don't need to expose any ports as since the script is making outbound requests, no inbound ports need to be exposed.